Example/Example/CharactersNavigationController.swift

```diff
@@ -13,7 +13,8 @@ class CharactersNavigationController: UINavigationController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        let viewController = AppDelegate.service.provideCharactersViewController()
-        pushViewController(viewController, animated: false)
+        let serviceLocator = ServiceLocator.SharedInstance
```

Comment:
We could add as another feature is the auto register of navigation service via a class.
And use a presenter to load the first ViewController.